pkg/binary/envoytest/util.go

```diff
@@ -15,46 +15,21 @@
 package envoytest
 
 import (
+	"bufio"
+	"bytes"
 	"context"
 	"io"
-	"log"
-	"os"
-	"path/filepath"
+	"strings"
 	"testing"
 	"time"
 
-	"github.com/mholt/archiver/v3"
 	"github.com/stretchr/testify/require"
 
 	"github.com/tetratelabs/getenvoy/pkg/binary/envoy"
-	"github.com/tetratelabs/getenvoy/pkg/globals"
-	"github.com/tetratelabs/getenvoy/pkg/test/morerequire"
 )
 
-// RunAndTerminateWithDebug is like RequireRunTerminate, except returns a directory populated by the debug plugin.
-func RunAndTerminateWithDebug(t *testing.T, debugDir string, debug func(r *envoy.Runtime) error, args ...string) string {
-	fakeEnvoy, removeFakeEnvoy := morerequire.RequireCaptureScript(t, "envoy")
-	defer removeFakeEnvoy()
-
-	fakeTimestamp := "1619574747231823000"
-	workingDir := filepath.Join(debugDir, fakeTimestamp)
-	fakeLogger := log.New(io.Discard, "", 0)
-	o := &globals.RunOpts{EnvoyPath: fakeEnvoy, WorkingDir: workingDir, Log: fakeLogger, DebugLog: fakeLogger}
-	// InitializeRunOpts creates this directory in a real command run
-	require.NoError(t, os.MkdirAll(o.WorkingDir, 0750))
-
-	r := envoy.NewRuntime(o)
-
-	e := debug(r)
-	require.NoError(t, e)
-
-	RequireRunTerminate(t, nil, r, args...)
-	RequireRestoreWorkingDir(t, o.WorkingDir, r)
-	return o.WorkingDir
-}
-
 // RequireRunTerminate executes Run on the given Runtime and terminates it after starting.
-func RequireRunTerminate(t *testing.T, terminate func(r *envoy.Runtime), r *envoy.Runtime, args ...string) {
+func RequireRunTerminate(t *testing.T, terminate func(r *envoy.Runtime), r *envoy.Runtime, args ...string) (err error) {
```

Comment:
Now binary/envoytest pkg is only used in binary/envoy so let's move the logic to binary/envoy/**_test.go and delete envoytest pkg 😄

Reply:
much simpler 😄

Reply:
It is also used by debug sadly..

Reply:
😞